.github/ISSUE_TEMPLATE/config.yml

```
@@ -1 +1,8 @@
 blank_issues_enabled: false
+contact_links:
+  - name: conda-store documentation
+    url: https://conda.store/en/latest/
+    about: Check out the conda-store documentation
+  - name: issues
+    about: Did you find an error? Report your findings here.
+    url: https://https://github.com/conda-incubator/conda-store/issues/new/choose
```

```suggestion
  - name: issues
    about: Did you find an error? Report your findings here.
    url: https://https://github.com/conda-incubator/conda-store/issues/new/choose
```

This only points to "bugs" and miss the feature and general issue type - these should be linked here directly so that all the templates show up together (again, see the nebari repo for an example)